modules/tinymce/CHANGELOG.md

```diff
@@ -78,6 +78,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Inserting a link for a selection from quickbars didn't preserve formatting. #TINY-9593
 - Inline dialog position was not correct when the editor wasn't inline and was contained in a `fixed` or `absolute` positioned element. #TINY-9554
 - Sticky toolbars would not have fade transition when undocking in classic iframe mode. #TINY-9408
+- Resolved a bug that caused inaccurate word counts for documents with specific characters. #TINY-8122
```

Comment:
We normally bug fix log messages like that. Compare to the other ones. Maybe this is better not sure:
```suggestion
- Word count was inaccurate for documents with specific characters. #TINY-8122
```
It is a bit generic but listing the characters seems tricky as well since they are so weird.

Reply:
yes we're in the "fixed" section of the changes, we don't need to include the phrase "resolved a bug" in each entry